apiserver/pkg/application/application.go

```diff
@@ -309,6 +309,12 @@ func ListApplicationMeatadatas(ctx context.Context, c dynamic.Interface, input g
 	}
 	totalCount := len(filtered)
 
+	if page < 0 {
+		page = 1
+	}
+	if pageSize < 0 {
```

Comment:
A function can be created to calculate the start and end positions of the paging.
@Abirdcfly @bjwswang Any suggestions?
```go
func PagePosition(page, size, total int) (int ,int) {
    start := (page-1)*size
    end :=  page * size
    if start > total {
        start = total
    }
    if end > total {
      end = total
    }
    return start, end
}
```

Reply:
Great if we have this common func.

Reply:
done